src/app/configureApp.jsx

```diff
@@ -132,14 +132,14 @@ const routeConfiguration = (siteConfig, componentConfig) => {
       tiny: {
         componentClass: ApplicationMenu,
         props: {
-          menuHeader: `${siteConfig.appConfig.title} ${siteConfig.appConfig.subtitle}`,
+          menuHeader: `${siteConfig.appConfig.title}`,
```

Comment:
The `appConfig.subtitle` option was removed in https://github.com/cerner/terra-site/pull/15 to align with  Application Name Components